sdk/src/main/java/io/opentelemetry/sdk/trace/RecordEventsReadableSpan.java

```diff
@@ -285,10 +293,9 @@ Status getStatus() {
    * @return An unmodifiable view of the attributes associated wit this span
    */
   @VisibleForTesting
```

Comment:
Can we remove the Visible for testing and always go via SpanData even in tests?

Reply:
We can, but I feel this is out of scope for this PR.